packages/aws-cdk-lib/aws-apigatewayv2/lib/websocket/integration.ts

```diff
@@ -24,7 +25,11 @@ export enum WebSocketIntegrationType {
   /**
    * Mock Integration Type
    */
-  MOCK = 'MOCK'
+  MOCK = 'MOCK',
+  /**
+   * AWS Integration Type
+   */
+  AWS = 'AWS'
```

Comment:
```suggestion
  AWS = 'AWS',
```

With the comma, next time, there's no need to diff the enum above

Reply:
Is there a formatter that we can use to automatically fix the format issues like trailing comm, whitespace, and etc?

Reply:
there is `prettier` but we don't use it for some reason in cdk. i'm sure it's come up before but i don't have the actual reason why.